### PR TITLE
Refs 3790: Folder restructure + absolute path update

### PR DESCRIPTION
**Pull request by Andrewgdewar**
## Summary

- Added support for absolute imports
- Updated folder structure for Pages
- Fixed a few rare test failures due to an older package dependency ("source-map"), a newer version of this package is now pinned in package.json.

## Testing steps
- Everything should just work, a quick sanity check is all that should be needed.

### CODE REVIEW

**Comment by jlsherrill** [comment]
https://issues.redhat.com/browse/HMS-3790

**Comment by jlsherrill** [comment]
⚠️ This task currently requires qe-approval, but this PR does not fully resolve the task.  Please contact QE to determine appropriate testing required.

File: src/components/SharedTables/AdvisoriesTable/index.tsx [hunk 1]
```diff
@@ -96,16 +96,22 @@ export default function AdvisoriesTable({
...
-                {columnHeaders.map(({ name, width }, index) => (
-                  name === 'Name' || name === 'Synopsis' ?  
-                  <Th width={width as BaseCellProps['width']} key={index + name + '_header'}>
-                    {name}
-                  </Th> :
-                  <Th width={width as BaseCellProps['width']} key={index + name + '_header'} sort={sortParams(index)}>
-                  {name}
-                </Th>
-                ))}
+                <Th screenReaderText='empty' />
```
Comment (xbhouse) [inline src/components/SharedTables/AdvisoriesTable/index.tsx:99]: i'm seeing this for this attribute: `Type '{ screenReaderText: string; }' is not assignable to type 'IntrinsicAttributes & Omit<ThProps, "ref"> & RefAttributes<HTMLTableHeaderCellElement>'.` was this intentional? or maybe i need to do an npm install?

Comment (xbhouse) [reply src/components/SharedTables/AdvisoriesTable/index.tsx:99]: everything seems to run ok despite this error

Comment (xbhouse) [reply src/components/SharedTables/AdvisoriesTable/index.tsx:99]: nevermind, npm install fixed that :)

**Review by xbhouse — APPROVED** [review]

**Comment by swadeley** [comment]
lgtm, UI tests pass locally

**Pull request merged**